pkgs/tools/misc/aquosctl/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+let
+  pname = "aquosctl";
+in
+stdenv.mkDerivation {
+  inherit pname;
+  version = "unstable-2014-04-06";
```

Comment:
```suggestion
stdenv.mkDerivation rec {
  pname = "aquosctl";
  version = "unstable-2014-04-06";
```

Reply:
I'd rather use a let binding than make the whole derivation recursive.

Reply:
Its mainly for readability and is used throughout nixpkgs. I tried to find a source for where its stated that its prefered, but ... I couldnt.

Reply:
```
❯ rg "inherit pname" | wc -l
3355
```

Reply:
The vast majority of those are used in fetchers (eg https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/games/cl-wordle/default.nix#L15)
```
❯ rg 'inherit pname' | wc -l
3354
❯ rg 'inherit pname' -B1 | rg fetch | wc -l
3011
```

.... but it doesnt really matter. Its just style.

(Also as a counterexample against myself)
```
❯ rg 'pname = "' | wc -l
45483
❯ rg 'pname = "' -B1 | rg rec | wc -l
16302
```
(Bad testing method? yes, but 16k is still a lot)